test/createDiscussionRepo.test.js

```diff
@@ -0,0 +1,14 @@
+const createDiscussionRepo = require('../lib/github-api/createDiscussionRepo')
+
+test('createDiscussionRepo is working', async () => {
```

Comment:
How about changing this test to use Jest's mock functions? Like this:

```js
const github = {
  repos: { createForOrg: jest.fn() }
}

await createDiscussionRepo(github, { owner })
expect(github.repos.createForOrg).toHaveBeenCalled()
expect(github.repos.createForOrg.mock.calls).toMatchSnapshot()
```

Reply:
Those are all the assertions you'd need. Here's the test I'm suggesting in full, with some extra comments:

```js
test('createDiscussionRepo is working', async () => {
  const github = {
    // Create a mock function
    repos: { createForOrg: jest.fn() }
  }

  // Call your function, which internally calls your above mock function
  await createDiscussionRepo(github, { owner: 'test' })

  // Test that your mock function has been called
  expect(github.repos.createForOrg).toHaveBeenCalled()

  // Test that your mock function has been called with the right arguments
  expect(github.repos.createForOrg.mock.calls).toMatchSnapshot()
})
```

For that last `expect`, if you'd prefer to use more explicit matching you can do this:

```js
expect(github.repos.createForOrg).toHaveBeenCalledWith({
  org: 'probot-background-check',
  name: 'test-discussions',
  description: 'Repo to have discussions about toxic users',
  // ... the rest of your arguments
})
```

Reply:
this is great, thanks `@jasonEtco`